src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -0,0 +1,20 @@
+package se.kth.sda.skeleton.comments;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Comment {
+
+    private Long id;
+    public void setId(Long id) {
+        this.id = id;
+    }
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Should these annotations be on the field, not the getter?

Reply:
Placed above the field id